plugin/toggleterm.vim

```diff
@@ -11,5 +11,6 @@ endif
 " Count is 0 by default
 command! -count -complete=shellcmd -nargs=* TermExec lua require'toggleterm'.exec_command(<q-args>, <count>)
 command! -count -nargs=* ToggleTerm lua require'toggleterm'.toggle_command(<q-args>, <count>)
-command! ToggleTermCloseAll lua require'toggleterm'.toggle_all("close")
-command! ToggleTermOpenAll lua require'toggleterm'.toggle_all("open")
+" command! ToggleTermCloseAll lua require'toggleterm'.toggle_all("close")
```

Comment:
@jorgerojas26 I was thinking that this and all the other code related to the old functionality should be deleted not just commented out, can you remove the commented code here and also the function they reference e.g. `toggle_all` should be replaced with your implementation so there is only one function doing this and only one command